DotNetifyLib.SignalR/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -27,7 +27,7 @@ namespace DotNetify
 {
    public static class ServiceCollectionExtensions
    {
-      public static IServiceCollection AddDotNetify(this IServiceCollection services)
+      public static IServiceCollection AddDotNetify(this IServiceCollection services, bool enableWebApi = true)
```

Comment:
Unit tests are failing because of this method signature change.  To fix, make this an overload and retain the original signature which in turn calls the overload:
```c#
public static IServiceCollection AddDotNetify(this IServiceCollection services) => services.AddDotNetify(true);
```

Reply:
Fixed. All tests are passed.

Reply:
fixed. all tests are passed